libs/MeshKernel/include/MeshKernel/Constants.hpp

```diff
@@ -41,11 +43,12 @@ namespace meshkernel
         // missing values
         namespace missing
         {
-            constexpr double innerOuterSeparator = -998.0;               ///< Double value used to separate the inner part of a polygon from its outer part
-            constexpr double doubleValue = -999.0;                       ///< Double value used as missing value
-            constexpr int intValue = -999;                               ///< Integer value used as missing value
-            constexpr UInt uintValue = std::numeric_limits<UInt>::max(); ///< missing value used for invalid indices
-        }                                                                // namespace missing
+            constexpr double innerOuterSeparator = -998.0;                                ///< Double value used to separate the inner part of a polygon from its outer part
+            constexpr double doubleValue = -999.0;                                        ///< Double value used as missing value
+            constexpr int intValue = -999;                                                ///< Integer value used as missing value
+            constexpr UInt uintValue = std::numeric_limits<UInt>::max();                  ///< missing value used for invalid indices
+            constexpr Eigen::Index EigenIndex = std::numeric_limits<Eigen::Index>::max(); ///< missing value used for invalid eigen indices
+        }                                                                                 // namespace missing
```

Comment:
Could you change the name of the constant EigenIndex? EigenIndex seems like a type name